### ISSUE TITLE
Forward rtp stream to multiple receivers

### ISSUE THREAD

**Issue by hupan-seu** [sippy/rtpproxy#116]
In some SIP cases, one SIP INVITE will be forked and sent to more than one receivers. After the SIP 183 code, the caller will send early media to the  receivers. So the early media need to be copied and sent to multiple receivers by Server, too. Does rtpproxy  support this? I saw the Recording/Copy control command(https://github.com/sippy/rtpproxy/wiki/RTPProxy-Command-Protocol), but there is not a clear example about how Copy command use. Could the Copy command make more client join the RTP session and receive the early media RTP data?

**Comment by sobomax** [comment]
Hi @hupan-seu. Such functionality is not currently supported unfortunately, however we have been receiving requests to make it possible several times over the past few years, so it's definitely on the drawing board. As such, I'll re-categorize your report as a "feature request" and keep it open until it's available.

Thanks for your interest and best regards.

**Comment by greenbea** [comment]
A typical scenario is sending a doorbell's early media to multiple receivers. I also would like to have this feature.

**Comment by sobomax** [comment]
@hupan-seu @greenbea yes, I was thinking about that as well. The idea there perhaps is to convert record/copy functionality into its own module and hook it into the packet processing using a new packet pipeline mechanism in 3.0. This would allow multiple independent "copy" sessions.  I will look into this for 3.1, hopefully.
